ManatiCR/manatime
# 19: Timesheet creation.

**Steps to test**

1.
 **Before test functionality**
- Check if the fields of timesheet content type were created fine.
- Check if the fields needed by timesheet content type were added to manatime_feature_field_bases.
- Check if the manatime_feature_timesheet was added to settings.php in the right order.
- Created some collaborators users.

2.
 **To test the functionality works**
- Run a variable_set in Execute PHP code of Drupal with a razonable date in timestamp format (It must be a date wich has already passed).
- Run cron.
- Check if the timesheet nodes corresponding to the collaborator users were created.

[Issue link](https://github.com/ManatiCR/manatime/issues/19)
